symplyphysics/laws/kinematic/rotational_inertia_cartesian_integral.py

```diff
@@ -0,0 +1,49 @@
+from sympy import Integral
+from symplyphysics import (
+    units,
+    Symbol,
+    Quantity,
+    Function,
+    validate_input,
+    validate_output,
+)
+
+# Description
+## In case of a rigid body with a continuously distributed mass, its rotational inertia is expressed
+## as an integral over the entire body.
```

Comment:
Mention somewhere, that it is volume integral, ie triple integral over x, y and z in cartesian coordinates.